src/assessments/pointer-motion/test-steps/motion-operation.tsx

```diff
@@ -24,8 +27,8 @@ const howToTest: JSX.Element = (
                     <li>Motion operation can be disabled by the user.</li>
                 </ol>
                 Exception: This requirement does not apply if motion activation is{' '}
-                <NewTabLink href="https://aka.ms/keros/essential">essential</NewTabLink> to the underlying function, such as tracking a
-                user’s steps.
+                <NewTabLink href="https://www.w3.org/TR/WCAG21/#dfn-essential">essential</NewTabLink> to the underlying function, such as
```

Comment:
Curious: Weren't these links stored somewhere in a common place to make the edits for these links much smaller?

Reply:
We can't use content links in test steps content as far as I remember, so no.